models/validator/schema.go

```diff
@@ -54,25 +109,18 @@ func IsValidCacaoJson(data []byte) error {
 
 	version := rawJson["spec_version"]
 
-	compiler := jsonschema.NewCompiler()
-	compiler.Draft = jsonschema.Draft7
-
-	var sch *jsonschema.Schema
-	var err error
 	switch version {
 	case cacao.CACAO_VERSION_1:
 		return errors.New("you submitted a cacao v1 playbook. at the moment, soarca only supports cacao v2 playbooks")
 	case cacao.CACAO_VERSION_2:
-		sch, err = compiler.Compile(oca_cacao_schemas)
-		if err != nil {
-			return err
+		schemaUrl := utils.GetEnv("VALIDATION_SCHEMA_URL", "")
+		if schemaUrl != "" {
+			return validateWithRemoteSchema(rawJson, schemaUrl)
+		} else {
+			return validateWithLocalSchema(rawJson)
```

Comment:
Should we perhaps also have an option "build local schemas", invoked once when building the code, which populates the schemas according to the latest version? Or something along those lines?